src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

```diff
@@ -180,7 +180,7 @@ To get started with API keys:
 
 You can view and manage most API keys from the [API keys UI page](https://one.newrelic.com/api-keys). To find that: from the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings/#user-menu), click <DoNotTranslate>**API keys**</DoNotTranslate>. You can also:
 
-* [Use NerdGraph explorer to view and create](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph#explorer) user keys
+* Use <InlinePopover type="nerdgraph" /> to view and create user keys
```

Comment:
@akristen I know you didn't compose these two bullets, but aren't they saying the same thing?

I had recently written a revamp of this page that moved this "Manage" section to the list in the introduction about managing keys. 

Also, you could move that reference to Account ID into the table of keys. There's no reason to have it orphaned like this.